### ISSUE TITLE
How do you use --idm actually?

### ISSUE THREAD

**Issue by ajam123** [justfoolingaround/animdl#13]
I use --idm but it doesn't seem to do anything and continuing download as normal via cmd.

Also, some episodes also being downloaded with `{number}`ts/s  and at the end, it only downloaded like 360p.

**Comment by justfoolingaround** [comment]
![image](https://user-images.githubusercontent.com/79979949/123360833-b349df00-d58d-11eb-9ab7-48e73f985f39.png)
Downloading with IDM seems to be working for me. Hence, I can't reproduce the issue you're having.

The download for HLS (that is, that `{number}`ts/s) you're getting is done at the highest quality. It's very likely that, that resolution is the source.

To solve the IDM issue, can you give me insights on:

- What provider you're using?
- What operating system is animdl running on?
- Optional: The anime name and episode you're downloading.

**Comment by ajam123** [comment]
- I'm using gogoanime provider, tried others same problem.
- I'm running on Window 10
- I'm trying to download Golden Kamuy 2nd Season.

With or without `--idm` same message:
[animdl-gogoanime-downloader-core] Initializing download session [00 -> ?]
[animdl-gogoanime-downloader-core] Starting download session [00 -> 11]
[animdl-gogoanime-downloader-core] Downloads will be done in the folder 'Tokyo ESP'

**Comment by justfoolingaround** [comment]
So, IDM Integration only works if IDM is installed to its standard directory;  which is expected to be `C:\Program Files (x86)\Internet Download Manager`.

Is IDM installed in that path? (Just give me a yes or a no, I'll look through fixes based on it if it's the cause!) 

(I'm sorry for the late response, was away for a bit.)

**Comment by ajam123** [comment]
Yes

My path `C:\Program Files (x86)\Internet Download Manager`

**Comment by justfoolingaround** [comment]
Aight, fixed, it should be working as intended now!

**Comment by ajam123** [comment]
It worked but seems like it doesn't download .ts extension?

**Comment by justfoolingaround** [comment]
Yeah, IDM doesn't support those extensions, so you'd just be getting like 1~3 kb m3u8 files if you download using it 😔

**Comment by justfoolingaround** [comment]
Aight, seems like the initial issue is solved, HLS takes place using animdl itself but don't worry, it'll give you the best quality possible. We can always re-open this issue if the problem persists.
